README.md

```diff
@@ -1,43 +1,49 @@
+<a href="https://www.twilio.com">
+  <img src="https://static0.twilio.com/marketing/bundles/marketing/img/logos/wordmark-red.svg" alt="Twilio" width="250" />
+</a>
+
 # Verify Quickstart
 
+![](https://github.com/TwilioDevEd/verify-v2-quickstart-python/workflows/Flask/badge.svg)
+
 > We are currently in the process of updating this sample template. If you are encountering any issues with the sample, please open an issue at [github.com/twilio-labs/code-exchange/issues](https://github.com/twilio-labs/code-exchange/issues) and we'll try to help you.
 
-Simple phone verification with Python, Flask, and Twilio Verify. 
+Simple phone verification with Python, Flask, and Twilio Verify.
 
-Full Quickstart instructions available at https://www.twilio.com/docs/verify/api-beta/quickstarts/python-flask
+[Read the full quickstart here](https://www.twilio.com/docs/verify/api-beta/quickstarts/python-flask)!
 
 ## Install
 
+    ```
     python3 -m venv venv
     . venv/bin/activate
+    ```
 
 Or on Windows cmd:
 
+    ```
     py -3 -m venv venv
     venv\Scripts\activate.bat
+    ```
 
-Install Verify:
+Install the dependencies:
 
+    ```
     pip install -r requirements.txt
+    ```
 
 ## Run
 
-    export FLASK_APP=verify
-    export FLASK_ENV=development
-    flask init-db
-    flask run
-
-Or on Windows cmd:
-
-    set FLASK_APP=verify
-    set FLASK_ENV=development
+    ```
     flask init-db
     flask run
```

Comment:
im getting `twilio.base.exceptions.TwilioRestException` when running the app and registering a new user

Reply:
I didn't get any. Did you get any specific message? Was the `VERIFICATION_SID` env variable correct? (I also included in the documentation now).

Reply:
@po5i turns out I had to manually `source .env` the env file. isn't that supposed to happen automatically? could we use a Makefile to contain that call along with the call that starts the service?

Reply:
Sure, I'll check out this. It's supposed for the env vars to be loaded automatically because we're using dotenv [here](https://github.com/TwilioDevEd/verify-v2-quickstart-python/blob/master/verify/__init__.py#L16). But maybe the location for this line is not appropriate. I'll test a clean clone.

Reply:
Confirmed. I did a clean clone in a clean zsh session, followed the steps and it worked as expected (user registration and phone verification successful).